docs/sphinx-docs/source/user/RELEASE.rst

```diff
@@ -13,6 +13,151 @@ Release Notes
+* Force main window to maximize #2272 by @rozyczko in #2273
+* A few more or less obvious fixes for speed of SasView startup by @rozyczko in #2275
+
+Plotting Fixes
+______________
+* fix for broken context menus in 1d plots by @rozyczko in #2670
+* DIfferentiate which combobox to put the graphs in. by @rozyczko in #2298
+* Fit intermittent plot blanking (#2267) by @pbeaucage in #2300
+* Fix unusable legend size with long filenames on Mac by @pbeaucage in #2264
+* Fix plot legend not updating on custom change. #2361 by @rozyczko in #2362
+* 2350: Close the BUMPS/DREAM results panel when data is deleted by @krzywon in #2365
+* fixed sesans residuals plots to show in real space rather than q space by @caitwolf in #2338
+
+Fitting and Other Perspective Fixes
+_____________
+* FIX: fix model date format in the user model docstring by @mrakitin in #2713
+* Various Multiplicity Model Fixes by @krzywon in #2647
+* 2855: Fix model save error by @krzywon in #2864
+* Fix "Use dQ Data" bug that swtiched slit length and slit width in the gui by @caitwolf in #2336
+* 2319: Fix copy/paste when selecting a structure factor by @krzywon in #2320
+* 1999: Clip fitting values set outside the fit range by @krzywon in #2422
+* Fixed breaking bug in pr_inversion by @lucas-wilkins in #2178
+* Fixes in invariant prespective by @wpotrzebowski in #2357
+* Fixes binning bug in implementation of sesans data by @caitwolf in #2331
+* fix confusing slit resolution parameters and add guardrails by @pbeaucage in #2283
+
+Other Fixes
+___________
+* Cherry-picked new_numpy_behaviour by @wpotrzebowski in #2655
+* Fit report save fix by @rozyczko in #2684
+* Fix for empty save format for grid files by @rozyczko in #2683
+
+Documentation Changes
+^^^^^^^^^^^^^^^^^^^^^
+* 2679: Update preferences documentation by @krzywon in #2680
+* Include SasData documentation in SasView by @krzywon in #2672
+* Corfunc Docs by @lucas-wilkins in #2823
+* Docstrings in perspective.py by @lucas-wilkins in #2207
+* Update Optimizer Help Doc by @smk78 in #2359
+* Update of the sld calculator documentation. Fixes #2689. by @katieweigandt in #2785
+* Update rst strings to fix doc build warnings by @smk78 in #2288
+
+Infrastructure Changes
+^^^^^^^^^^^^^^^^^^^^^^
+
+Build System
+____________
+* Rework CI to be non-repetitive, simpler, test more things by @llimeht in #2263
+* Stop existing concurrent CI by @krzywon in #2638
+* Release 6.0.0 notarization fix by @wpotrzebowski in #2709
+* Fixes problem with using deprecated node.js by @butlerpd in #2836
+* Run unit tests of Qt GUI and have CI gate on them by @llimeht in #2252
+* Remove ubuntu 18.04 from CI by @krzywon in #2439
+
+Python Dependencies
+___________________
+* Added 3.11, removed 3.8 by @rozyczko in #2582
+* Warnings removals and python 3.9 drop in tests by @wpotrzebowski in #2860
+* Pyside6 merge by @rozyczko in #2478
+* Pyinstaller 6 by @krzywon in #2854
```

Comment:
This should say Change GUI package from PyQt5 to Pyside6. Also, This actually a big deal right? - no doubt some breaking changes and part of why this  is 6 not 5.1?